scipio/src/executor.rs

```diff
@@ -362,6 +373,8 @@ impl LocalExecutorBuilder {
         let mut le = LocalExecutor::new(EXECUTOR_ID.fetch_add(1, Ordering::Relaxed));
         if let Some(cpu) = self.binding {
             le.bind_to_cpu(cpu)?;
+        } else {
```

Comment:
I don't understand why this is being done only if there is no binding.
It's actually the contrary: we should only allow spinning at all if there is binding: if there is no binding you may have two executors in the same CPU, now fighting each other by spinning like crazy.

Reply:
I misinterpret this:

> In our implementation, an executor may or may not be pinned to a CPU:
> Executors that are pinned should never poll.
> Executors that are pinned may or may not poll, depending on the configuration.

I still don't understand what it means.

Reply:
Sorry Ivan, this is my bad.
Notice how both phrases start with "Executors that are pinned" so one of them has to be wrong =)
Executors that are *not* pinned should never poll.
Executors that are pinned may or may not poll, depending on the configuration.